clientcontroller/interface.go

```diff
@@ -45,8 +45,8 @@ type ClientController interface {
 	// QueryFinalityProviderVotingPower queries the voting power of the finality provider at a given height
 	QueryFinalityProviderVotingPower(fpPk *btcec.PublicKey, blockHeight uint64) (uint64, error)
 
-	// QueryFinalityProviderSlashed queries if the finality provider is slashed
-	QueryFinalityProviderSlashed(fpPk *btcec.PublicKey) (bool, error)
+	// QueryFinalityProviderSlashedOrJailed queries if the finality provider is slashed or jailed
+	QueryFinalityProviderSlashedOrJailed(fpPk *btcec.PublicKey) (bool, bool, error)
```

Comment:
suggestion to add names to return when they are not explicitly 
```suggestion
	QueryFinalityProviderSlashedOrJailed(fpPk *btcec.PublicKey) (slashed bool, jailed bool, err error)
```